keywords/TestServerNetMsft.py

```diff
@@ -18,7 +18,31 @@ def __init__(self, version_build, host, port, community_enabled=None, platform="
 
         if version_build <= "2.1.0":
             raise Exception("No .net based app available to download for 2.1.0 or below at latestbuild. Use nuget package to create app.")
-        if self.platform == "net-msft":
+
+        if self.platform == "c-msft":
+            if community_enabled:
```

Comment:
as more type of .net apps available, I would suggest to use the current Factory design pattern to fetch a single class instead of more if-then conditions. while, it's optional but cleaner way for coding.

Reply:
Create another class for c windows ?